### DISCUSSION TITLE
Welcome to spring-comparing-template-engines Discussions!

### DISCUSSION THREAD

**Discussion by jreijn** [jreijn/spring-comparing-template-engines#62] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Vest** [comment]
Hi @jreijn 

Do you have any ideas or future plans regarding this project? :)
Should we drop Java 8 (and old libraries) and switch to Java 11?

**Reply by jreijn** [reply]
I've been thinking about that. I would assume that most developers that find this project are already working with or experimenting with Java 11+. Even though most of the reports indicate that lots of people are still on Java 8 at work.

**Reply by Vest** [reply]
that's true.
